[Bug]: Linker errors

### Category

Other

### Hardware

Heltec Mesh Node T114

### Firmware Version

2.5.20.c1beb44

### Description

I've just started getting linker errors attempting to build on Windows 10. It seems to be a result of [#5790 platformio.ini](https://github.com/meshtastic/firmware/pull/5790/files#diff-4446afd728a4f34cbcddc306a9cb6be845d1a61c216076a295683bcc9c106724).

### Relevant log output

```Shell
Linking .pio\build\heltec-mesh-node-t114\firmware.elf
c:/users/todd/.platformio/packages/toolchain-gccarmnoneeabi/bin/../lib/gcc/arm-none-eabi/9.3.1/../../../../arm-none-eabi/bin/ld.exe: cannot open map file D:DevelopmentProjectsMeshtasticgeneralmeshtastic-firmware.piobuild/output.map: No such file or directory      
collect2.exe: error: ld returned 1 exit status
*** [.pio\build\heltec-mesh-node-t114\firmware.elf] Error 1
```

--- a/platformio.ini
+++ b/platformio.ini
@@ -20,7 +20,7 @@ extra_scripts = bin/platformio-custom.py
 build_flags = -Wno-missing-field-initializers
 
 	-Wno-format
-	-Isrc -Isrc/mesh -Isrc/mesh/generated -Isrc/gps -Isrc/buzz -Wl,-Map,${platformio.build_dir}/output.map
+	-Isrc -Isrc/mesh -Isrc/mesh/generated -Isrc/gps -Isrc/buzz -Wl,-Map,"${platformio.build_dir}"/output.map
 	-DUSE_THREAD_NAMES
 	-DTINYGPS_OPTION_NO_CUSTOM_FIELDS
 	-DPB_ENABLE_MALLOC=1

